src/main/java/com/dev/cinema/Main.java

```diff
@@ -66,32 +70,32 @@ public static void main(String[] args) throws AuthenticationException {
         authenticationService.register(johny.getEmail(), johny.getPassword());
         johny = authenticationService.login(johny.getEmail(), johny.getPassword());
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block